special-interest-groups/sig-planner/README.md

```diff
@@ -0,0 +1,33 @@
+# Planner Special Interest Group(Planner SIG)
+
+Planner SIG covers TiDB planner/statistics implementations. We are focused on
+the development and maintenance of the TiDB [planner](https://pingcap.com/docs/stable/reference/performance/sql-optimizer-overview/)
+and [statistics](https://pingcap.com/docs/stable/reference/performance/statistics/) component, including but not limited to:
+
+* Supporting new features.
+* Improving the performance.
+* Fixing bugs.
+
+## Roles and Organization Management
+
+See [Planner SIG Roles and Organization Management](./roles-and-organization-management.md)
+
+## Members
+
+See [SIG Planner Member List](./member-list.md)
+
+## Meetings
+
+* Regular Working Group Meeting: 5:00, Thursday at PT (Pacific Time) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=5:00&tz=PT%20%28Pacific%20Time%29).
+* Meeting Zoom: [ttps://pingcap.zoom.com.cn/j/492702291](ttps://pingcap.zoom.com.cn/j/492702291)
+* Meeting Notes: to be updated
```

Comment:
please add a public meeting notes doc.

Reply:
Do we need to fill it before first meeting is held?

Reply:
I think a meeting note link is ok, even there is no content for now.